### DISCUSSION TITLE
Why did not pass here

### DISCUSSION THREAD

**Discussion by acodercat** [prisma/prisma#6493] [Q&A] [answered]
```ts
this.prismaService.realTimeMonitoringData.findMany({
      where: {
        datetime: '2021-04-05 19:17:20',
      }
});
```

it prompts me:
```bash
Argument datetime: Got invalid value '2021-04-05 19:17:20' on prisma.findManyRealTimeMonitoringData. Provided String, expected DateTimeFilter or DateTime:
type DateTimeFilter {
  equals?: DateTime
  in?: List<DateTime>
  notIn?: List<DateTime>
  lt?: DateTime
  lte?: DateTime
  gt?: DateTime
  gte?: DateTime
  not?: DateTime | NestedDateTimeFilter
}
```
RealTimeMonitoringDataWhereInput:

```ts


  export type RealTimeMonitoringDataWhereInput = {
    AND?: Enumerable<RealTimeMonitoringDataWhereInput>
    OR?: Enumerable<RealTimeMonitoringDataWhereInput>
    NOT?: Enumerable<RealTimeMonitoringDataWhereInput>
    id?: IntFilter | number
    data?: JsonNullableFilter
    monitoring_station_id?: IntFilter | number
    base_id?: IntFilter | number
    base?: XOR<BaseRelationFilter, BaseWhereInput>
    area?: XOR<AreaRelationFilter, AreaWhereInput>
    monitoring_station?: XOR<MonitoringStationRelationFilter, MonitoringStationWhereInput>
    area_id?: IntFilter | number
    datetime?: DateTimeFilter | Date | string
  }
```

**Comment by ryands17** [comment, marked as answer]
@acodercat 👋 

You need to pass an ISO 8601 Extended Format in the form of a string so you need to use the [toISOString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) method on your date and pass that.

**Reply by acodercat** [reply]
Thank you!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
